test/functional/fixtures/regression/gh-3049/test.js

```diff
@@ -20,11 +19,11 @@ if (isLocalChrome) {
                         .src(path.join(__dirname, './testcafe-fixtures/index.js'))
                         .run();
                 })
-                .then(err => {
+                .then(failedCount => {
                     testCafe.close();
 
-                    if (err)
-                        throw new Error('Error occured');
+                    if (failedCount)
+                        throw new Error('Error is occurred');
```

Comment:
The `Error is occurred` message is not correct.

Reply:
I will apply this change after Andrey review it.